loadtestconfig.rst

```diff
@@ -0,0 +1,571 @@
+
+
+    -example    ``"MattermostInstallDir":/home/christopher/go/src/github.com/mattermost/mattermost-server/dist/mattermost``
+
+ConfigFileLoc 
+-------------------------------------
+
+    ``ConfigFileLoc`` is the location of the Mattermost configuration file.
+    If this user variable is not empty it is passed to the Mattermost binary as the --config parameter.
+    
+    -example    ``"ConfigFileLoc":/home/christopher/go/src/github.com/mattermost/mattermost-server/dist/mattermost/config``
+
+AdminEmail 
+-------------------------------------
+
+      ``AdminEmail`` is the email address of the Administrator account on the server. It is created if it does not already exist.
+
+    -example    ``"AdminEmail":success+ltadmin@simulator.amazonses.com``
+
+AdminPassword 
+-------------------------------------
+
+    ``AdminPassword`` is the password for the Administrator account.
+
+    -example    ``"AdminPassword":ltpassword``
+
+SkipBulkload 
+-------------------------------------
+
+    ``SkipBulkload`` can be set to true to save time with verification if you are running the load test multiple times and have already loaded all the users into the database.
+
+
+    -options    true/false
+    -default    false
+    -example    ``"SkipBulkload":false``
+
+WaitForServerStart 
+-------------------------------------
+
+    ``WaitForServerStart`` decides whether to wait for the server to start before connecting.
+    
+    -options    true/false
+    -default    false
+    -example    ``"WaitForServerStart":false``
+
+ 
+
+ 
+Loadtest Enviroment Configuration
+=========================================================
+
+The Loadtest Enviroment Configuration section contains all the variables for setting up the enviroment.
+In the Mattermost lexicon enviroment means the load limits and timing of communications. 
+This includes the number of teams, channels, users, posts, percentages, channel selections, 
+and other things.
+
+The variables are as follow below.
+
+
+NumTeams
+----------------------------------------------------
+
+    ``NumTeams`` is the number of teams you want for the test.
+    
+    -example    ``"NumTeams":1,``
+
+
+
+
+NumChannelsPerTeam
+-----------------------------------------------------------------------------
+
+    ``NumChannelsPerTeam`` is the number of channels allocated to each team.
+    
+    -example      ``"NumChannelsPerTeam":400,``
+    
+
+
+
+NumUsers 
+----------------------------------
+
+    ``NumUsers`` is the number of users in the load test.
+
+    -example    ``"NumUsers":1000,``
+
+NumPosts 
+-----------------------------------------------------------------------------
+
+    ``NumPosts`` is the number of posts in the load test.
+    
+        -example    ``"NumPosts":20000000,``
+
+
+PostTimeRange
+-----------------------------------------------------------------------------
+
+    ``PostTimeRange`` is the range of time allocated to a post in milliseconds.
```

Comment:
Unclear. Maybe: "Post time range is the oldest a generated post will be from the current time in milliseconds. This only applies to loaded posts, not to posts during a loadtest."